Gemfile

```diff
@@ -84,7 +84,7 @@ gem "omniauth-salesforce", "~> 1.0"
 gem "omniauth-rails_csrf_protection"
 
 # Added at 2017-10-24 00:03:53 -0700 by dkaplan:
-gem "restforce", "~> 3.1"
+gem "restforce", "~> 4.2"
```

Comment:
Updated both of these and QA'd through the app to make sure general functionality is still as expected